test/e2e/autoscaler_test.go

```diff
@@ -20,7 +20,6 @@ package e2e
 
```

Comment:
Can you comment that files in here can run with Chaosduck and the criteria how to pick one way or another?